docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -119,32 +116,31 @@ The following commands are accepted by the API:
 
    .. code-block:: cpp
 
-      /*** With Inference Engine 2.0 API ***/
+      /*** With OpenVINO Runtime API 2.0 ***/
       ov::Core core; 
 
       // Read a network in IR, PaddlePaddle, or ONNX format:
-      std::shared_ptr<ov::Model> model = core.read_model("sample.xml");    
+      std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
```

Comment:
move these snippets to .cpp file

Reply:
@kblaszczak-intel  do you agree to move snippets to .cpp and .py ?

Reply:
I guess w should, since this is the new approach. In the future, we need to work on organizing these files, though. Thanks @tiger100256-hu for dealing with snippets here.

Reply:
do we need to demonstrate old API 1.0? In other documents we use only API 2.0